ams/server/src/main/java/com/netease/arctic/server/manager/FlinkOptimizerContainer.java

```diff
@@ -183,9 +183,15 @@ protected String buildOptimizerStartupArgsString(Resource resource) {
     resourceFlinkConf.putToOptions(
         FlinkConfKeys.TASK_MANAGER_TOTAL_PROCESS_MEMORY, taskManagerMemory + "m");
 
+    // Set memory managed fraction
+    resourceFlinkConf.putToOptions(
+        FlinkConfKeys.MANAGED_MEMORY_FRACTION, FlinkConfKeys.MANAGED_MEMORY_FRACTION_DEFAULT);
```

Comment:
We'd better not include irrelevant changes in the PR. 

Also, in the docs, we recommend this configuration [flink-conf.taskmanager.memory.managed.size=32mb](https://github.com/NetEase/amoro/blob/ca2061edcb117b472eb9e0a3c84933c37118b79e/docs/admin-guides/managing-optimizers.md?plain=1#L56)